### DISCUSSION TITLE
extend 'extupdates' with historic information of updates

### DISCUSSION THREAD

**Discussion by jborgman1432** [akeeba/panopticon#734] [Ideas]
Sometimes, when debugging, it would be very useful to see when an update is installed.
I think it would be nice to extend the information on each line on /index.php?view=extupdates with the install dates of the previous versions.

**Comment by nikosdion** [comment]
You mean what we already have in [Site Reports](https://github.com/akeeba/panopticon/wiki/Site-Reports) for this exact reason?

**Comment by jborgman1432** [comment]
yes, something like that. In Site Reports it also shows a lot of line for every time an update has been found by the update schedule. Probably adding a filter to this view so only installation actions are shown will result in my idea.

**Reply by nikosdion** [reply]
Adding filters on that view is definitely something I have in mind :)
